config/initializers/dev_and_test_environment.rb

```diff
@@ -0,0 +1,3 @@
+unless Rails.env.production?
+  ENV['HOST'] = 'hwproj.me'
+end
```

Comment:
Speaking on the file name I think this file is needed only for setting a host and doesn't really set up dev/test environment.
BTW, why did you decide not to place this config in `application.rb` as I suggested?